File: Dataset/cpp/marilyn.cpp

//  marilyn.cpp  (c) 1997-2002 Kari Laitinen

#include  <iostream.h>
#include  <string.h>

int main()
{
   char  name_to_be_known[]  =  "Marilyn Monroe" ;
   char  name_from_keyboard[ 40 ] ;

   cout <<  "\n Who played a main role in movies:"
        <<  "\n\n   \"How To Marry a Millionaire\" (1953)"
        <<    "\n   \"The Seven Year Itch\" (1955)"
        <<    "\n   \"The Misfits\" (1961) \n\n   ? " ;

   cin.getline( name_from_keyboard, sizeof( name_from_keyboard ) ) ;

   if ( strcmp( name_from_keyboard, name_to_be_known ) == 0 )
   {
      cout  <<  "\n   Yes, that is correct.\n" ;
   }
   else
   {
      cout  <<  "\n   No, it\'s Marilyn Monroe.\n" ;
   }
}

File: Dataset/cs/Marilyn.cs

//  Marilyn.cs (c) 2004 Kari Laitinen

//  www.naturalprogramming.com

//  04.06.2003  C# file created.
//  06.07.2004  Last modification

using System ;

class Marilyn
{
   static void Main()
   {
      string  name_to_be_known  =  "Marilyn Monroe" ;

      Console.Write( "\n Who played the main role in the movies:"
                  +  "\n\n   \"How To Marry a Millionaire\" (1953)"
                    +  "\n   \"The Seven Year Itch\" (1955)"
                    +  "\n   \"The Misfits\" (1961) \n\n   ? " ) ;

      string  name_from_keyboard  =  Console.ReadLine() ;

      if ( String.Compare( name_from_keyboard,
                           name_to_be_known ) == 0 )
      {
         Console.Write( "\n   Yes, that is correct.\n" ) ;
      }
      else
      {
         Console.Write( "\n   No, it\'s Marilyn Monroe.\n" ) ;
      }
   }
}



